### ISSUE TITLE
Updating embedding input sizes in document tutorials

### ISSUE THREAD

**Issue by hojinYang** [tensorflow/recommenders#92]
Dear all, thanks for the great project and tutorial examples! It helped me figure out how tensorflow-recommender works. To be of little help, I just created pull requests #85 about the input dimension of the embedding matrix in the examples in the docs. 

For timestamp embedding, I fixed ` tf.keras.layers.Embedding(len(timestamp_buckets) + 2, 32)` to `tf.keras.layers.Embedding(len(timestamp_buckets) + 1, 32)`. The index ranges from 0 to the size of the timestamp_buckets. That is, if the number of buckets is 1,000, we only need 1000 + 1 indexes, not 1000 + 2.

For user embedding, I fixed ` tf.keras.layers.Embedding(len(unique_user_ids) + 2, embedding_dimension)` to `tf.keras.layers.Embedding(len(unique_user_ids) + 1, embedding_dimension)`, as mask embedding is never used in two-tower model. I set `mask_token` argument of `StringLookup` to `None` to ignore mask embedding. Same as item embedding. 

Maybe it does not have anything to do with the tutorial itself, as the original version also works. If so, please excuse me (I am just trying to be helpful and make the embedding size more clearly).
